src/modules/project/pages1/projectDashboard/views/promote/AmbassadorPayoutSections.tsx

```diff
@@ -0,0 +1,150 @@
+}
+
+/** AmbassadorPayoutsSection: Displays the Ambassador Payouts table and add form */
+export const AmbassadorPayoutsSection = ({
+  ambassadors,
+  ambassadorsLoading,
+  ambassadorTableSchema,
+  handleAddAmbassadorSubmit,
+  onAddAmbassador,
+  registerAddAmbassador,
+  addAmbassadorErrors,
+  addAmbassadorLoading,
+}: AmbassadorPayoutsSectionProps) => {
+  return (
+    <CardLayout w="full">
+      <VStack spacing={4} align="start">
+        <Body size="xl" medium>
+          {t('Ambassador Payouts')}
+        </Body>
+        <Body size="sm" light>
+          {t(
+            'Ambassadors spread the word about your project. You can reward your most loyal ambassadors by sharing a percentage of the contributions they enable with them.',
+          )}
+        </Body>
+
+        {/* Improved Add Ambassador Form */}
+        <form
+          id={ADD_AMBASSADOR_FORM_ID}
+          onSubmit={handleAddAmbassadorSubmit(onAddAmbassador)}
+          style={{ width: '100%' }}
+        >
+          {/* Use SimpleGrid for better responsive layout */}
+          <SimpleGrid columns={{ base: 1, md: 4 }} spacing={4} alignItems="start" mb={4}>
+            {/* Hero ID Input */}
+            <FormControl isInvalid={Boolean(addAmbassadorErrors.heroId)} gridColumn={{ md: 'span 2' }}>
+              <Input
+                size="sm"
+                placeholder={t('User Hero ID')}
+                {...registerAddAmbassador('heroId', { required: t('User ID is required') })}
+                isDisabled={addAmbassadorLoading}
+              />
+              <FormErrorMessage>{addAmbassadorErrors.heroId && addAmbassadorErrors.heroId.message}</FormErrorMessage>
+            </FormControl>
+
+            {/* Payout Rate Input */}
+            <FormControl isInvalid={Boolean(addAmbassadorErrors.payoutRate)}>
+              <InputGroup size="sm">
+                <Input
+                  type="number"
+                  placeholder={t('Rate')}
+                  step="0.01"
+                  {...registerAddAmbassador('payoutRate', {
+                    required: t('Rate is required'),
+                    min: {
+                      value: PAYOUT_RATE_MIN,
+                      message: t('Rate must be > {{min}} and <= {{max}}', { min: 0, max: PAYOUT_RATE_MAX }),
+                    },
+                    max: {
+                      value: PAYOUT_RATE_MAX,
+                      message: t('Rate must be > {{min}} and <= {{max}}', { min: 0, max: PAYOUT_RATE_MAX }),
+                    },
+                    valueAsNumber: true,
+                  })}
+                  textAlign="right"
+                  isDisabled={addAmbassadorLoading}
+                />
+                <InputRightElement>%</InputRightElement>
+              </InputGroup>
+              <FormErrorMessage>
+                {addAmbassadorErrors.payoutRate && addAmbassadorErrors.payoutRate.message}
+              </FormErrorMessage>
+            </FormControl>
```

Comment:
For cases like these, where there is a bit more nuanced UI with input group is required, we can still standardize the labels and errors with : 

FieldContainer component (src/shared/components/form/FieldContainer.tsx).

that is the same one that wraps up the ControlledInputs. 

Also for validations, we create a separate schema with YUP, and provide it to the useForm itself.

This separates the form logic from the UI components, and keeps it cleaner.